extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts

```diff
@@ -291,13 +282,14 @@ export class GmailElementReplacer extends WebmailElementReplacer {
     return !!$('iframe.pgp_block').filter(':visible').length;
   };
 
-  private addMenuButton = (action: 'reply' | 'forward', selector: string) => {
-    const gmailActionsMenuContainer = $(this.sel.msgActionsMenu).find(selector);
-    const button = $(this.factory.actionsMenuBtn(action)).insertAfter(gmailActionsMenuContainer); // xss-safe-factory
-    button.on(
-      'click',
-      Ui.event.handle((el, ev: JQuery.Event) => this.actionActivateSecureReplyHandler(el, ev))
-    );
+  private addMenuButton = (replyOption: ReplyOption, gmailContextMenuBtn: string) => {
+    if ($(gmailContextMenuBtn).css('display') === 'block') {
```

Comment:
this can be replaced by `if ($(gmailContextMenuBtn).is(':visible')) {`

Reply:
Got it - @sosnovsky, I'll change it as suggested. Btw, could you please confirm if it works for you as you expected it to work. For me, it works very well now, very unnoticeable to almost no latency.

Reply:
It works almost perfect now, one thing left is for the test to pass. If you noticed issues along the way, please let me know.